src/css/styles.scss

```diff
@@ -1,46 +0,0 @@
-// Load CSS Reset from NPM
-// CSS resets "normalize" how browser styles are applied.
-// Try uncommenting this and saving. What changes in the browser?
-@import "~normalize.css";
-
-// An example of a partial.
-// Check out the _typography.scss file to edit the CSS inside!
-@import "typography";
-@import "layout";
```

Comment:
These two are optional. If you decide not to keep them, delete these lines too.

However, if you want to keep your CSS files separate, like welcome.css, you could move it to the `src/css` folder and rename it `_welcome.scss`. Then, write `@import "welcome";` here. That will keep your styles nice and neatly organized, and you won't have to copy paste them all here. This is called a partial in Sass.